stylesheets/abstract/variables.css

```diff
@@ -3,4 +3,8 @@
   --red-alert: red;
   --light-green-alert: rgb(122, 248, 139);
   --green-alert: green;
+  --highlight: #f9e506;
+
+  --hover-bg: rgb(255, 0, 68);
+  --hover: #51fb42;
```

Comment:
Try to use same type of variables, don't mix between solids, RGB and #.

Reply:
ok done as u suggested